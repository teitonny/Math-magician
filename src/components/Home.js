import HomeCss from './Home.module.css';

const Home = () => (
  <div className={HomeCss.homeContainer}>
    <h1> Welcome to the calculate</h1>
    <p>
      {' '}
      Lorem ipsum dolor sit amet.
      Et quaerat blanditiis in voluptate atque
      eos eveniet praesentium est enim sequi qui reiciendis facere.
      Est odio tenetur est earum odit ea odit voluptatum qui alias
      corrupti est delectus vero ut voluptatem tenetur hic
      aspernatur corporis. Ea omnis aspernatur est
      similique quia est dolorum quia. In rerum iusto
      non rerum optio aut perspiciatis iure quo dolore provident
      et accusamus odit aut fuga aperiam At voluptas accusantium
    </p>
    <p>
      {' '}
      Lorem ipsum dolor sit amet.
      Et quaerat blanditiis in voluptate atque
      eos eveniet praesentium est enim sequi qui reiciendis facere.
      Est odio tenetur est earum odit ea odit voluptatum qui alias
      corrupti est delectus vero ut voluptatem tenetur hic
      aspernatur corporis. Ea omnis aspernatur est
      similique quia est dolorum quia. In rerum iusto
      non rerum optio aut perspiciatis iure quo dolore provident
      et accusamus odit aut fuga aperiam At voluptas accusantium
    </p>
    <p>
      {' '}
      Lorem ipsum dolor sit amet.
      Et quaerat blanditiis in voluptate atque
      eos eveniet praesentium est enim sequi qui reiciendis facere.
      Est odio tenetur est earum odit ea odit voluptatum qui alias
      corrupti est delectus vero ut voluptatem tenetur hic
      aspernatur corporis. Ea omnis aspernatur est
      similique quia est dolorum quia. In rerum iusto
      non rerum optio aut perspiciatis iure quo dolore provident
      et accusamus odit aut fuga aperiam At voluptas accusantium
    </p>
  </div>
);

export default Home;
