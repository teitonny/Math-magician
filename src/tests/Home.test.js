import React from 'react';
import Renderer from 'react-test-renderer';
import Home from '../components/Home';

it('renders correctly', () => {
  const tree = Renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
