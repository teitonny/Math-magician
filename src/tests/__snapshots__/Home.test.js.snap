// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="homeContainer"
>
  <h1>
     Welcome to the calculate
  </h1>
  <p>
     
    Lorem ipsum dolor sit amet. Et quaerat blanditiis in voluptate atque eos eveniet praesentium est enim sequi qui reiciendis facere. Est odio tenetur est earum odit ea odit voluptatum qui alias corrupti est delectus vero ut voluptatem tenetur hic aspernatur corporis. Ea omnis aspernatur est similique quia est dolorum quia. In rerum iusto non rerum optio aut perspiciatis iure quo dolore provident et accusamus odit aut fuga aperiam At voluptas accusantium
  </p>
  <p>
     
    Lorem ipsum dolor sit amet. Et quaerat blanditiis in voluptate atque eos eveniet praesentium est enim sequi qui reiciendis facere. Est odio tenetur est earum odit ea odit voluptatum qui alias corrupti est delectus vero ut voluptatem tenetur hic aspernatur corporis. Ea omnis aspernatur est similique quia est dolorum quia. In rerum iusto non rerum optio aut perspiciatis iure quo dolore provident et accusamus odit aut fuga aperiam At voluptas accusantium
  </p>
  <p>
     
    Lorem ipsum dolor sit amet. Et quaerat blanditiis in voluptate atque eos eveniet praesentium est enim sequi qui reiciendis facere. Est odio tenetur est earum odit ea odit voluptatum qui alias corrupti est delectus vero ut voluptatem tenetur hic aspernatur corporis. Ea omnis aspernatur est similique quia est dolorum quia. In rerum iusto non rerum optio aut perspiciatis iure quo dolore provident et accusamus odit aut fuga aperiam At voluptas accusantium
  </p>
</div>
`;
